index.Rmd

```diff
@@ -75,3 +79,7 @@ https://github.com/fhdsl/DaSEH
 This page was last updated on `r Sys.Date()`.
 
 <p style="text-align:center;"><a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://live.staticflickr.com/4557/26350808799_6f9c8bcaa2_b.jpg" height="150"/> </a> </p><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License</a>.
```

Comment:
Can we center the text on this one too?

Reply:
sure no problem

Reply:
wait it already is... maybe it just looks funny